src/components/demos/write/LoginForm.js

```diff
@@ -1,27 +1,23 @@
 import React, { useState, useEffect } from 'react'
-import styled from 'styled-components'
 
 import Input from '../../shared/Input'
 import { Container, Spinner } from '../../shared/Common'
 import { extractLoginResult } from '../../../utils/write-demo-output'
-
-const Button = styled.button((props) => props)
```

Comment:
a nice change, I didn't realis this could just be replaced with `<button {...styling}>` although it seems so obvious now. There's a few other places in the repo with `((props) => props)` that could be replaced at some point in a PR...

Reply:
yea, this is where we went a little beyond and we started passin some DOM nodes